### ISSUE TITLE
Define username and password during deployment via docker-compose.yml 

### ISSUE THREAD

**Issue by sprnza** [swarmpit/swarmpit#608]
Hi there!
Thank you for the project!
Is there a way to define username and password in swarmpit's ENV variables in order to eliminate necessity to add user and private registry manually?

**Comment by nguni52** [comment]
When we run our swarmpit via docker we are able to inject a user in the following manner:
docker run --rm  --name swarmpit-installer  --volume /var/run/docker.sock:/var/run/docker.sock -e INTERACTIVE=0 -e ADMIN_USERNAME=xxx -e ADMIN_PASSWORD=xxxx swarmpit/install:1.9

**Comment by nohaapav** [comment]
@nguni52 Yes this was possible in version 1.9 and older .. in latest version this was removed.
